src/Forms/FixCommonErrors/FixOcrErrosViaReplaceList.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Nikse.SubtitleEdit.Core;
+using Nikse.SubtitleEdit.Core.Forms.FixCommonErrors;
+using Nikse.SubtitleEdit.Logic.Ocr;
+using System.Windows.Forms;
+
+namespace Nikse.SubtitleEdit.Core.Forms.FixCommonErrors
+{
+    public class FixOcrErrorsViaReplaceList : IFixCommonError
+    {
+        public string ThreeLettersISOLanguageName { get; set; }
+        public Form ParentForm { get; set; }
+
+        public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
+        {
+            using (var ocrFixEngine = new OcrFixEngine(ThreeLettersISOLanguageName, null, ParentForm))
+            {
+                var language = Configuration.Settings.Language.FixCommonErrors;
+                var fixAction = language.FixCommonOcrErrors;
+                int noOfFixes = 0;
+                string lastLine = string.Empty;
+                for (int i = 0; i < subtitle.Paragraphs.Count; i++)
+                {
+                    var p = subtitle.Paragraphs[i];
+                    string text = ocrFixEngine.FixOcrErrors(p.Text, i, lastLine, false, OcrFixEngine.AutoGuessLevel.Cautious);
+                    lastLine = text;
+                    if (callbacks.AllowFix(p, fixAction) && p.Text != text)
+                    {
+                        string oldText = p.Text;
+                        p.Text = text;
+                        noOfFixes++;
+                        callbacks.AddFixToListView(p, fixAction, oldText, p.Text);
+                    }
+                    Application.DoEvents();
+                }
+                if (noOfFixes > 0)
+                {
+                    callbacks.AddToTotalErrors(noOfFixes);
```

Comment:
Mistake I know... but I think there should a method called AddTotalFixes in `IFixCallBacks` to track total fixes., right?